vector/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you move and rename this file to the same location and spirit than https://github.com/vector-im/element-android/blob/main/library/ui-styles/src/main/res/values/stylable_badge_floating_action_button.xml please?

Reply:
Okay I see. We declare one file per view. I will do the update.